src/App.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
 import './App.scss';
+import Sum from './components/Sum';
+
+const input = [[2, 2], [10, 0], [-5, 5]];
 
 const App = () => (
-  <h1>React sum</h1>
+  <React.Fragment>
+    {
+      input.map(item => (
+        <Sum
```

Comment:
if you map items, you should use `key`
https://reactjs.org/docs/lists-and-keys.html